corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import absolute_import
```

Comment:
had to enable absolute imports because there's an xml.py in the casexml/apps/phone directory which conflicts with the ElementTree import. all imports in this file are aboslute